src/main/java/com/google/sps/servlets/schedule/util/Time.java

```diff
@@ -0,0 +1,158 @@
+package com.google.sps.servlets;
+
+import java.text.SimpleDateFormat;
+import java.time.ZoneId;
+import java.util.GregorianCalendar;
+import java.time.format.DateTimeFormatter;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.TimeZone;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.joda.time.DateTimeZone;
+import org.joda.time.DateTime;
+
+/** Should be able to READ string. */
+public class Time {
+    // For recurrences!
+    public final String SUNDAY = "SU";
```

Comment:
Maybe we can declare a Map<Integer, String> where the key = 1 to 7 mapped to "SU", "MO", "TU" so on.

You can initialize the map when you create it: https://stackoverflow.com/questions/6802483/how-to-directly-initialize-a-hashmap-in-a-literal-way

Then you can replace the case statement with a map lookup.

Reply:
Makes sense!